pulp_container/app/registry_api.py

```diff
@@ -613,12 +645,31 @@ def put(self, request, path, pk=None):
             except IntegrityError:
                 pass
 
-            with repository.new_version() as new_version:
-                new_version.add_content(models.Blob.objects.filter(pk=blob.pk))
-
             upload.delete()
 
-            return BlobResponse(blob, path, 201, request)
+            task = enqueue_with_reservation(
```

Comment:
i get occasionally this error `AttributeError: 'Task' object has no attribute 'id'`
we should not use `task` as variable here, but rather `job` because on line 663 `task` variable holds a task object and not Job, and so if hitting continue statement on line 668 will make us looping through the cycle and we'd fail on line 663

Reply:
Ha! So you are saying, if the thing takes longer than the first second, we start to hit this?